pkg/nsx/services/vpc/vpc.go

```diff
@@ -51,15 +51,15 @@ type VPCService struct {
 	IpblockStore *IPBlockStore
 }
 
-func (s *VPCService) RegisterVPCNetworkConfig(ncCRName string, info VPCNetworkConfigInfo) {
+func (s *VPCService) RegisterVPCNetworkConfig(ncCRName string, info common.VPCNetworkConfigInfo) {
```

Comment:
It's good to put some comments to say this implementation of which interface function.